libraries/version/src/version.cpp

```diff
@@ -12,4 +12,9 @@ namespace eosio { namespace version {
       return version;
    }
 
+   const std::string& version_hash() {
+      static const std::string vhash{_version_hash()};
```

Comment:
Seems like an odd thing to keep around for the life of the application. Looks like a copy paste from above, but the same thing goes for those as well.

Reply:
Yeah I was just following how it was done before, it does seem unnecessary

Reply:
@heifner is refactoring libversion something you want to see done?

Reply:
Not a huge priority, so this is fine for now.